telegram/_chat.py

```diff
@@ -790,6 +790,136 @@ async def set_administrator_custom_title(
             api_kwargs=api_kwargs,
         )
 
+    async def set_photo(
+        self,
+        photo: FileInput,
+        *,
+        read_timeout: ODVInput[float] = DEFAULT_NONE,
+        write_timeout: ODVInput[float] = 20,
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: JSONDict = None,
+    ) -> bool:
+        """Shortcut for::
+
+             await bot.set_chat_photo(
+                 update.effective_chat.id, *args, **kwargs
+             )
+
+        For the documentation of the arguments, please see
+        :meth:`telegram.Bot.set_chat_photo`.
+
```

Comment:
```suggestion

        .. versionadded:: 20.0

```

please also add this to all other methods that you added